### PR TITLE
feat: Use `codacy-analysis-cli` in an action [CY-2806]

### PR DESCRIPTION
**Pull request by afsmeira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+
+The example file above uses all the default configurations.
+
+## Configuration
+
+You can configure this action by changing the following settings in the workflow `.yml` file.
+
+|Setting |Description |Required |Default value |
+|--- |--- |--- |--- |
+|`project-token` |The project API token |Yes |N/A |
```
Comment (afsmeira) [inline README.md:34]: I'm not sure which other settings to support. I think it makes sense to support `tool` (but I'm confused about how it works) and maybe `api-token`, `username` and `project`.

**Comment by afsmeira** [comment]
Should I add a license file?

File: README.md (hunk 1 above)
Comment (rtfpessoa) [reply README.md:34]: Why not support all of then? I think eventually we will need them and they are not here.
Form example, for sarif I will need `--output` and `--format`

**Comment by rtfpessoa** [comment]
> Should I add a license file?

I think it would be nice. Most of our repos have apache 2

**Comment by rtfpessoa** [comment]
I just got this idea but not sure if it makes sense. Could this yaml be in the cli repo?
Having the action colocated with the tool seems nice.

**Comment by afsmeira** [comment]
> I just got this idea but not sure if it makes sense. Could this yaml be in the cli repo?
> Having the action colocated with the tool seems nice.

Yeah, I think that makes more sense. That way, when you update the tool, you can update the action easily. I will reopen this in the correct repo and delete this one. 👍

**Pull request closed without merging**